kie-pmml-trusty/kie-pmml-evaluator/kie-pmml-evaluator-assembler/src/main/java/org/kie/pmml/evaluator/assembler/implementations/HasKnowledgeBuilderImpl.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kie.pmml.evaluator.assembler.implementations;
+
+import org.drools.compiler.builder.impl.KnowledgeBuilderImpl;
+import org.kie.internal.builder.KnowledgeBuilder;
+import org.kie.pmml.kie.dependencies.HasKnowledgeBuilder;
+
+public class HasKnowledgeBuilderImpl implements HasKnowledgeBuilder {
+
+    private final KnowledgeBuilderImpl knowledgeBuilder;
```

Comment:
If you use `KnowledgeBuilder` instead of `KnowledgeBuilderImpl` you can remove the dependency to `drools.compiler` and move this `HasKnowledgeBuilderImpl` to `kie-pmml-kie-dependencies` directly (and get rid of all the `*Mock` impl you created). Wdty?

```suggestion
    private final KnowledgeBuilder knowledgeBuilder;
```